File: leets/1078/1078.py
class Solution(object):
    def findOcurrences(self, text, first, second):
        """
        :type text: str
        :type first: str
        :type second: str
        :rtype: List[str]
        """
        arrs = text.split(' ')
        length = len(arrs)
        result = []
        for index in range(0, length - 2):
            if arrs[index] == first and arrs[index + 1] == second:
                result.append(arrs[index + 2])
                index += 2
        return result

File: leets/1078/1078.go
package main

import (
	"fmt"
	"strings"
)

func FindOcurrences(text string, first string, second string) []string {
	var result []string
	arrs := strings.Split(text, " ")
	for i := 0; i < len(arrs)-2; i++ {
		if arrs[i] == first && arrs[i+1] == second {
			result = append(result, arrs[i+2])
		}
	}
	return result
}

func main() {
	text := "we will we will rock you"
	first := "we"
	second := "will"
	fmt.Print(FindOcurrences(text, first, second))
}
